### DISCUSSION TITLE
How can I show WakaTime stats on my GitHub README? (Easiest way)

### DISCUSSION THREAD

**Discussion by DevHanza** [community/community#145354] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I'm trying to add my WakaTime stats to my GitHub README.md file to showcase my coding activity. Does anyone know a simple and effective way to do this? 

I've seen some people using it but I'm not sure which approach is best. Any tips or suggestions would be greatly appreciated!

**Comment by dalvarez2596** [comment, marked as answer]
Hi @DevHanza 
Im not using it, but I found this repo which instructions to achieve that https://github.com/anmol098/waka-readme-stats, hope this work.
As I was reading you need your waka api key.

**Reply by DevHanza** [reply]
I saw it too, but it seems like a lot of work to set up. I'm looking for a quick solution with a good UI. Anyways, Thanks for the reply.

**Comment by mecodeatlas** [comment]
Hi @DevHanza 👋🏾,It looks like @dalvavarez2596's response was the answer to your question here. Can you please mark their response as the answer? When you mark a question as an answer, GitHub will highlight the comment and replies to the comment to help visitors quickly find the answer.
![image](https://github.com/user-attachments/assets/b89e7161-c2d1-47e2-a058-242df633e6b2)

**Comment by DevHanza** [comment]
For anyone who is looking for an answer,
I found this repo which has great documentation and better options.
https://github.com/athul/waka-readme
